docs/docs/architecture-guides/.pages

```diff
@@ -0,0 +1,4 @@
+nav:
+  - index.md
+  - chainblocks.md
+  - ...
```

Comment:
It isn't really necessary: by default `index.md` is always first followed by all pages in alphabetical order. It is mostly useful when the order should be different.

It's not a mistake to have it though.

Reply:
Ok, let's leave it as it for now.